src/Utils.ts

```diff
@@ -97,12 +97,22 @@ export function findEventInRoom(client: MatrixClient, roomId: string, eventId: s
 
 // @internal
 export function buildTextMessage(event: MatrixEvent, status: MessageStatus): TextMessage {
-    return {
-        text: event.getContent().body,
-        timestamp: event.getTs(),
-        sender: event.getSender(),
-        status: status,
-        id: event.getId()
+    if (event.event.type === 'org.decentraland.friendship') {
```

Comment:
I need to include this conditional statement because the message body is stored in the `message` attribute for the `org.decentraland.friendship` event, but it is stored in the `body` attribute for the `m.room.message` event. I think it would be a good idea to refactor the code and consistently use the body attribute for both types of events. What do you think?

Reply:
The last [commit](https://github.com/decentraland/matrix-client/pull/116/commits/66a22bdc68b2edbd1d027ea7fd000993a4bc4680) includes a refactor to consistently use the 'body' attribute for both types of events.